contrib/bump_version.bash

```diff
@@ -85,16 +98,19 @@ then
 	BODY="Changelog since $LAST_TAGGED_VERSION:
 $BODY"
 
-	git tag -a "$NEW_VERSION" -m "$BODY"
+    git tag -a "$NEW_VERSION" -m "$BODY"
 
-	echo "pushing to git..."
-	git push upstream && git push upstream $NEW_VERSION
+    echo "creating pull request in github..."
+    git push -u origin "Release-$NEW_VERSION"
+    hub pull-request -m "$JIRA_TICKET: Release $NEW_VERSION" --base mongodb:master --head mongodb:"Release-$NEW_VERSION"
 else
-	set +e
-	git commit -m "Update $NEW_VERSION"
-	if [ $? -eq 0 ]; then
-		set -e
-		git push upstream
+    set +e
+    git commit -m "$JIRA_TICKET: Update $NEW_VERSION"
+    if [ $? -eq 0 ]; then
+	    echo "creating pull request in github..."
+        set -e
+        git push -u origin "Release-$NEW_VERSION"
+        hub pull-request -m "$JIRA_TICKET: Release $NEW_VERSION" --base mongodb:master --head mongodb:"Release-$NEW_VERSION"
 	fi
 	set -e
 fi
```

Comment:
[minor] I think everything below this line should be in a separate script (`contirb/publish_sdk.sh`) that should only be run after the pull request is merged. We shouldn't be publishing anything to bintray before it hits master. When you move this stuff to a new script, the $NEW_VERSION variables will have to be replaced with the current version, which you can fetch the same way as how you fetch LAST_VERSION in this script. ask me in person if you have any questions